### ISSUE TITLE
blast

### ISSUE THREAD

**Issue by pavithraavula** [koalaman/shellcheck#2830]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash
query_seqs=$1
genome_assembly=$2
outfile=$3

tblastn \
        -query "$query_seqs" \
        -subject "$genome_assembly" \
        -outfmt '6 std qlen' \
        | awk '$3>30 && $4>0.9*$13' \
        | cut -f9,10 > "$outfile"

bed=$4
genes=$5
while read -r qstart qend
do
        awk "$qstart >= $(cut -f2 "$bed") && $qend <= $(cut -f2 "$bed")" > "$genes"
        wc -l < "$genes"
done < "$outfile"
```

#### Here's what shellcheck currently says:
```
99680
```
#### Here's what I wanted or expected to see:
26
TODO: Describe expected/desired output

**Comment by brother** [comment]
Retitle and expand with more information before proposing a reopen of this.
